csv-validator-core/src/main/scala/uk/gov/nationalarchives/csv/validator/FailFastMetaDataValidator.scala

```diff
@@ -60,16 +60,10 @@ trait FailFastMetaDataValidator extends MetaDataValidator {
     validateRows()
   }
 
-  private def validateRow(row: Row, schema: Schema): MetaDataValidation[Any] = {
```

Comment:
Some of these methods where duplicated in the base class and the AllErrorsMetaDataValidator class. So to avoid duplicate code, i refactored some methods visibility into protected instead of private, and overrode the subclass implementation when needed.